lib/expectations_runner.rb

```diff
@@ -19,9 +25,18 @@ class Mumukit::Inspection::PlainInspection
 
   def expectations
     {
-      'HasWhile' => /AST\(while/
+      'HasWhile' => use(/AST\(while/),
```

Comment:
@AlvarezAriel this is the kind of stuff you like, perhaps you have already tried it